src/main/scss/components/_tabs.scss

```diff
@@ -86,3 +86,108 @@ $border-radius: 19px;
   margin: 2.2rem 0 1.4rem;
   padding: 0;
 }
+
+.new-tabs {
```

Comment:
All of these classes need to be pulled out into their appropriate files, renamed and tidied up.